### DISCUSSION TITLE
Highlight message in chat

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#26042] [Ideas] [answered]
After you got mentioned, you should see your username/word that mentions you at first glance (green is good)

**Comment by Gabixel** [comment]
(Apparently, a chat highlight feature is already implemented (#17078, #17088, PR: #17137), but I didn't notice it yet in my experience, or maybe yes but it was hardly visible)

**Reply by bdach** [reply]
The feature in question briefly flashes the message containing the mention. I'm assuming OP wants the actual text mentioning them _inside_ the message to be highlighted.

**Comment by cl8n** [comment]
I noticed in the daily challenge today that finding the messages that highlighted me was kind of difficult. would be nice if either my name within their message was given a special color, or if the whole message was given a different bg color to stand out

**Reply by bdach** [reply]
it already is given a different bg colour, but it might be an issue of that marker disappearing too quickly...?

**Reply by cl8n** [reply]
oh I must have missed it. I guess what I'm requesting for is a permanent style or indicator of some kind

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/29314
